docs/database/sqlalchemy.rst

```diff
@@ -10,7 +10,7 @@ for more information.
 
 Alternatively, you can try to follow
 :ref:`wiki tutorial <bfg_sql_wiki_tutorial>` or
-`blogr tutorial <http://pyramid-blogr.readthedocs.org>`_.
+`blogr tutorial <https://pyramid-blogr.readthedocs.io>`_.
```

Comment:
The canonical host name for all Pylons Projects is `docs.pylonsproject.org`. The correct URL in this case is http://docs.pylonsproject.org/projects/pyramid-blogr/en/latest/